### PR TITLE
Create 2018-11-05-connexion-20-release.md

### PR DESCRIPTION
**Pull request by jmcs**
## Description

Documents the release of Connexion 2.0.

## Types of Changes

- Blog Post

## Tasks
- [x] Release Connexion 2.0

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply:_

- [ ] My change requires a change to the documentation & I have updated it accordingly.

### CODE REVIEW

**Comment by baggerspion** [comment]
@jmcs Don't forget to sign-off your commits! If you are planning to push a few more commits to this branch, the easiest thing is to simply push them all unsigned and then squash and then sign the single commit at the end. Otherwise, we cannot merge this.

**Comment by jmcs** [comment]
@therealpadams I signed off now.

File: _posts/2018-11-05-connexion-20-release.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+(formerly known as Swagger Spec) of your API described in YAML format. Connexion allows you to write a Swagger specification, 
+then maps the endpoints to your Python functions.
+
+Besides routing, Connexion also validates requests and responses automatically based on OpenAPI specifications, handles common
+authentication schemes, supports API versioning and supports automatic serialization of payloads. It can use both
+[Flask](http://flask.pocoo.org/) and [aiohttp](https://github.com/aio-libs/aiohttp) as backend servers.
+
+Besides OpenAPI 3 support, this release includes a more streamlined internal structure, better adherence to Swagger 2.0 spec by
+default, and support for basic authentication and apikey authentication. For a more detailed list of changes, check
+[Connexion's Read Me](https://github.com/zalando/connexion/tree/dev-2.0#new-in-connexion-20).
```
Comment (hjacobs) [inline _posts/2018-11-05-connexion-20-release.md]: I guess the README link should point to `master` now and not the `dev-2.0` branch?

**Review by hjacobs — APPROVED** [review]

**Comment by hjacobs** [comment]
LGTM

**Comment by hjacobs** [comment]
Not sure whether we need a new header image for the blog post?

**Comment by hjacobs** [comment]
@jmcs I guess you need to add the "connected" image to git first?

**Review by hjacobs — APPROVED** [review]

**Review by baggerspion — APPROVED** [review]

**Pull request merged**